### PR TITLE
style: Upload loading icon should be primary color

### LINKED ISSUE #39112: Upload loading icon should be primary color
图片

https://ant.design/components/upload-cn#components-upload-demo-picture-style

### PR DESCRIPTION
**Pull request by MARKX97**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

close #39112

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Change Upload primary color       |
| 🇨🇳 Chinese |     增加upload默认颜色       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/upload/style/picture.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ const genPictureStyle: GenerateStyle<UploadToken> = (token) => {
 
             [iconCls]: {
               fontSize: token.fontSizeHeading2,
+              color: token.blue,
```
Comment (afc163) [inline components/upload/style/picture.tsx:36]:
```suggestion
              color: token.colorPrimary,
```

File: components/upload/style/picture.tsx [hunk 2]
```diff
@@ -79,6 +80,8 @@ const genPictureStyle: GenerateStyle<UploadToken> = (token) => {
 
 const genPictureCardStyle: GenerateStyle<UploadToken> = (token) => {
   const { componentCls, iconCls, fontSizeLG, colorTextLightSolid } = token;
+  console.log('token', token);
```
Comment (afc163) [inline components/upload/style/picture.tsx:83]: Should not add this

Comment (MARKX97) [reply components/upload/style/picture.tsx]: fixed

File: components/upload/style/picture.tsx (hunk 1 above)
Comment (MARKX97) [reply components/upload/style/picture.tsx]: fixed

**Review by afc163 — APPROVED** [review]

**Pull request merged**